File: crescente.py
#Leia uma quantidade indeterminada de duplas de valores inteiros X e Y. Escreva para cada X e Y uma
#mensagem que indique se estes valores foram digitados em ordem crescente ou decrescente. O
#programa deve finalizar quando forem digitados dois valores iguais.

print("Digite dois números: ")
x = int(input())
y = int(input())

while x != y:
   if x < y:
       print("Crescente.")
   else:
       print("Decrescente.")
   print("Digite dois números: ")
   x = int(input())
   y = int(input())

File: workSpace Java/curso_programaçao/src/crescente.java
import java.util.Scanner;
public class crescente {

	public static void main(String[] args) {
	   
	   Scanner sc = new Scanner(System.in);
	   
	   int x, y;
	   
	   System.out.println("Digite dois numeros: ");
	   x = sc.nextInt();
	   y = sc.nextInt();
       
	   while (x != y){
	   
	      if (x < y) {
		     System.out.println("Crescente.");
	      }
	      else {
		     System.out.println("Decrescente.");
	      }
	      
	      System.out.println("Digite dois numeros: ");
		   x = sc.nextInt();
		   y = sc.nextInt();
		   
	   }
	   sc.close();
	}

}
